src/chocolatey/infrastructure.app/configuration/ConfigurationOptions.cs

```diff
@@ -120,6 +123,12 @@ public static OptionSet OptionSet
 
             if (configuration.HelpRequested)
             {
+                if (configuration.ShowOnlineHelp)
+                {
+                    System.Diagnostics.Process.Start(string.Format("https://docs.chocolatey.org/en-us/choco/commands/{0}", configuration.CommandName));

```

Comment:
Just relying on the command name could be a problem (I think), unless we also redirect aliased command names to the correct page on the docs site

Reply:
It may make sense to create ch0.co links? That way if the docs site changes, the link still can work.

Reply:
@AdmiringWorm we have pages in place for all the command aliases, you can see them listed here:

* https://github.com/chocolatey/docs/tree/master/input/en-us/choco/commands
* https://github.com/chocolatey/docs/tree/master/input/en-us/create/commands

So I think we are covered there.

@TheCakeIsNaOH I actually had the same thought, but I wasn't sure if it was warranted here or not.  Happy to add them though.  What would be the suggestion here?  Something like:

* https://ch0.co/c/<command name>

Reply:
@gep13 that formatting/syntax for the short link looks good,